<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CirclesMainWidget</class>
 <widget class="QWidget" name="CirclesMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="algoSelector">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Polar Co-ordinate</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bresenham's Midpoint</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cartesian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Midpoint</string>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>479</width>
     <height>479</height>
    </rect>
   </property>
   <widget class="PolarCircleWidget" name="page_0"/>
   <widget class="BresenhamMidpointCircleWidget" name="page_1"/>
   <widget class="CartesianCircleWidget" name="page_2"/>
   <widget class="MidpointCircleWidget" name="page_3"/>
  </widget>
  <widget class="QSpinBox" name="cx">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="cy">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="r">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>300</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="gridWidth">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>80</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
  </widget>
  <widget class="QPushButton" name="drawButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>360</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>450</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>572</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Grid Width</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="timeDisplay">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ForbiddenCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>63</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Radius</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PolarCircleWidget</class>
   <extends>QWidget</extends>
   <header>circles/polarcirclewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BresenhamMidpointCircleWidget</class>
   <extends>QWidget</extends>
   <header>circles/bresenhammidpointcirclewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CartesianCircleWidget</class>
   <extends>QWidget</extends>
   <header>circles/cartesiancirclewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MidpointCircleWidget</class>
   <extends>QWidget</extends>
   <header>circles/midpointcirclewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>gridWidth</tabstop>
  <tabstop>algoSelector</tabstop>
  <tabstop>cx</tabstop>
  <tabstop>cy</tabstop>
  <tabstop>r</tabstop>
  <tabstop>drawButton</tabstop>
  <tabstop>timeDisplay</tabstop>
  <tabstop>resetButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
